
bool ESTOP
bool driveModeLed
bool robotechError
bool GPIOInitalized
float64 motorVoltage
float64 motorCurrent
float64 temperature
float64 batteryVoltage